Stripe/NSString+Stripe.m

```diff
@@ -28,6 +35,29 @@ - (NSString *)stp_reversedString {
     return [mutableReversedString copy];
 }
 
+- (NSString *)stp_stringByRemovingCharactersFromSet:(NSCharacterSet *)cs {
```

Comment:
This could maybe be simplified by just using NSMutableString's `deleteCharactersInRange` in a loop?  Although I guess you are just moving existing code around so it's fine.